.github/workflows/e2e.yaml

```diff
@@ -56,9 +79,12 @@ jobs:
         with:
           name: ci-images
 
-      - name: Import the KMMO image into minikube
+      - name: Import the KMM operator image into minikube
         run: minikube image load kmm_local.tar
 
+      - name: Copy the signing image to the registry
+        run: skopeo copy --dest-tls-verify=false tarball:kmm-signimage.tar docker://localhost:5000/kmm/signimage:latest
```

Comment:
Because that would insert the image into the minikube node, and the signing job can't pull it - it needs to be in a registry accessible from the pods.